velox/core/QueryConfig.h

```diff
@@ -342,6 +342,9 @@ class QueryConfig {
   static constexpr const char* kSparkLegacyStatisticalAggregate =
       "spark.legacy_statistical_aggregate";
 
+  static constexpr const char* kSparkLegacyFollowThreeValuedLogicInArrayExists =
+      "spark.follow_three_valued_logic_in_array_exists";
```

Comment:
Would you please document this config in https://github.com/facebookincubator/velox/blob/main/velox/docs/configs.rst#spark-specific-configuration?